<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1275</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1275</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1271</width>
      <height>781</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Single</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>741</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Test Parameters</string>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>691</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>PZT Coordinates</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>30</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>z</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>y</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="singleX">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>31000</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="singleY">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>30</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>93000</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="singleZ">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>30</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>60300</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="singleDelta">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>30</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>800</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>30</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Delta</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>691</width>
         <height>181</height>
        </rect>
       </property>
       <property name="title">
        <string>Other Parameters</string>
       </property>
       <widget class="QLineEdit" name="singleV1">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>60</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Compress Vel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>30</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Cycles</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="singleCycles">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>30</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="xyzPort">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>30</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ASRL3::INSTR</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>XYZ Port</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Compress Time</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="singleDelay1">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>120</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>60</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Lift Velocity</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="singleV2">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>60</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>120</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Relax Time</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="singleDelay2">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>120</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Sampling Rate</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="singleFreq">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>150</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>3000</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="singleDcc">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>90</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Acceleration</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>90</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Deccelleration</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="singleAcc">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>90</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bringHome">
        <property name="geometry">
         <rect>
          <x>519</x>
          <y>150</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
       <widget class="QPushButton" name="startSingle">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>150</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>1251</width>
        <height>441</height>
       </rect>
      </property>
      <property name="title">
       <string>Results</string>
      </property>
      <widget class="PlotWidget" name="forceGraph" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>601</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
      <widget class="PlotWidget" name="voltageGraph" native="true">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>50</y>
         <width>621</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="calibrationFactor">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>20</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Delay</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Calibration Factor</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rangeRegen">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>20</y>
         <width>321</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Regenerate Range</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>20</y>
        <width>481</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Result Summary</string>
      </property>
      <widget class="QPushButton" name="exportSingle">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>260</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
      <widget class="QLabel" name="avgRatio">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="projD33">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>90</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="PlotWidget" name="d33Box" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>241</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="stdRatio">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>60</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="stdRatio_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>60</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>SD:</string>
       </property>
      </widget>
      <widget class="QLabel" name="projD33_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>90</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Projected d33:</string>
       </property>
      </widget>
      <widget class="QLabel" name="avgRatio_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Average Ratio:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Array</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Calibrate</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>951</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>PZT Coordinates</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>30</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>z</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="calibrateX">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>31000</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="calibrateY">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>93000</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="calibrateZ">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>30</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>60300</string>
       </property>
      </widget>
      <widget class="QPushButton" name="calibrateMove">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>70</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Move</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>V</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="calibrateA">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>70</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>20</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="calibrateD">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>70</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>20</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>70</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>70</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>A</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="calibrateV">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>50</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>30</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Delay</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="calibrateDelay">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>30</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="calibratePort">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ASRL3::INSTR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bringHome_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1275</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControl_Center">
    <property name="title">
     <string>NPI Force Hammer Control</string>
    </property>
   </widget>
   <addaction name="menuControl_Center"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
